### ISSUE TITLE
Formular  3.4.3.4.1 F Eingangskontrolle OG nachdem Ticket 385 programmiert

### ISSUE THREAD

**Issue by Toelk-A** [chrbusch/GTM-V#385]

**Comment by Toelk-A** [comment]
Anbei das Dokument. 

[Githubticket_385___ 3.4.3.4.1 F Eingangskontrolle OG-05-11.08.17.docx](https://github.com/chrbusch/GTM-V/files/2145128/Githubticket_385___.3.4.3.4.1.F.Eingangskontrolle.OG-05-11.08.17.docx)

**Comment by Toelk-A** [comment]
Bitte heute mit Frau Schröder besprechen und programmieren

**Comment by chrbusch** [comment]
Die Druckmaske und das zugeh. Script sind nun auch bald fertig, auch dies könnte nun getestet werden

**Comment by chrbusch** [comment]
Wir müssen besprechen, was "Mitarbeiter Herstellung" genau meint.
Ist das Rolle 9, "nicht-ärztl. Mitarbeiter" bei GTMV? Dann wären es die Personen im Bild

![bildschirmfoto 2018-08-26 um 08 56 23](https://user-images.githubusercontent.com/1241899/44625619-f2531400-a90d-11e8-9b2c-44479c3392a8.JPG)

**Comment by Toelk-A** [comment]
Mitarbeiter Herstellung ist IMMER GBM-V:

Widmer
Hohmann
Schröder, Emma
Schmidt
Schmitt
Koß
Schottmann
Deuschle

**Comment by chrbusch** [comment]
Die Mitarb.-Liste ist jetzt m.E. korrekt. Das Dokument kann ab sofort getestet werden, was Frau Schröder auch heute (280818) tun wird

**Comment by Toelk-A** [comment]
Transplantatdetails - linkes Hauptmenü- Button "Eingangskontrolle und Tastenkombination Strg+5 kommt folgende Fehlermeldung

![image](https://user-images.githubusercontent.com/22314271/44847311-5d288600-ac53-11e8-80a1-1c3691fb437b.png)

**Comment by chrbusch** [comment]
Fehler bestätigt, verstanden und behoben

**Comment by Toelk-A** [comment]
Ich benötige Daten zum Testen des Formulars.

**Comment by EmmaSchroederGBMV** [comment]
Ich habe die ID´s 2912 - 2912 bzw HH 2018-0605 bis HH 2018-0608 jetzt mit den Eingangskontrollen gefüllt. Reicht das zum testen?

**Comment by Toelk-A** [comment]
Bitte die folgenden Markierungen programmieren/ändern

[Github 385_Eingangskontrolle_Teil 1.pdf](https://github.com/chrbusch/GTM-V/files/2419178/Github.385_Eingangskontrolle_Teil.1.pdf)

**Comment by chrbusch** [comment]
(TransplantatID 2912)
L+R sind nun vorhanden; ging vorher schon bei Klick auf den Button "Eingangskontrolle" links in dr Liste, nun auch bei den Prozessschritten.)
Dig. US ist da.
Die fehlenden Daten wurden m.E. einfach nicht eingegeben!

**Comment by Toelk-A** [comment]
## Die fehlenden Daten wurden m.E. einfach nicht eingegeben!
Emma kannst du dies bitte prüfen?
Danke

**Comment by EmmaSchroederGBMV** [comment]
Ja, das stimmt aber das haben wir ja schon besprochen. Das "S" ist auch vorhanden.
Mir ist allerdings aufgefallen das die Maske für die Eintragungen sehr schön ist nur sobald das Protokoll über die Prozessschritte geöffnet wird ist es viel zu klein. 
![eingangskontrolle_gesamt](https://user-images.githubusercontent.com/43231378/46134514-576a9400-c243-11e8-81c0-d2fdba9b428f.png)

**Comment by Toelk-A** [comment]
Transplantat ID 2912

## 1) PDF bildet nicht S + D gemeinsam ab
Öffnet man die Eingangskontrolle mit der Eingabemaske sind S(links) und D(rechts) gemeinsam abgebildet. Öffnet man die zugehörige PDF(Prozesschritte -> Eingangskontrolle -> Strg+5) so ist nur "D" abgebildet. Gehts man zur Partnerhornhaut "S" und öffnet die PDF ist auch nur "D" abgebildet.

Bitte für den speziellen Fall prüfen.

**Comment by chrbusch** [comment]
Problem verstanden und behoben, bitte testen

**Comment by Toelk-A** [comment]
Es funktioniert noch immer nicht-.---- klickt man auf CS wird im PDF trotzdem CD abgebildet...

**Comment by chrbusch** [comment]
NUN habe ich verstanden, dass das Druckscript, was i.d.R. sinnvoll ist, immer nur einen Datensatz druckt.
Für die Eing.Ktr. habe ich hiervon nun eine Ausnahme programmiert, und bei mir klappt es.

![bildschirmfoto 2018-11-05 um 10 42 26](https://user-images.githubusercontent.com/1241899/47990343-96fc8980-e0e7-11e8-98f9-08f70fc4b65c.JPG)

**Comment by EmmaSchroederGBMV** [comment]
Hallo, bezüglich der Eingangskontrolle gehen die Unterschriften von Anne Theel (TSA), Nicole Schottmann (SCN) und Anna Reinelt (REA) noch nicht.

**Comment by chrbusch** [comment]
Bitte präziser formulieren: Was genau klappt nicht?

**Comment by chrbusch** [comment]
..ich she aber gerade, dass bei diesen Mitarbeitern noch gar keine Unterschriften hinterlegt sind!
Wenn Sie mir digitalisierte Unterschriften als JPG schicken, kann ich diese einsetzen; ich denke aber, Frau Tölk kann das auch ohne mich
